Makefile

```diff
@@ -0,0 +1,66 @@
+VENV := $(shell echo $${VIRTUAL_ENV-.venv})
+INSTALL_STAMP := $(VENV)/.install.stamp
+PSQL_INSTALLED := $(shell psql --version 2>/dev/null)
+
+clean:
+	find . -name '*.pyc' -delete
+	find . -name '__pycache__' -type d | xargs rm -rf
+
+distclean: clean
+	rm -rf *.egg *.egg-info/ dist/ build/
+
+maintainer-clean: distclean
+	rm -rf .venv/
+
+$(INSTALL_STAMP): requirements.txt requirements-dev.txt
+	virtualenv $(VENV) --python=python3
+	$(VENV)/bin/python -m pip install --upgrade pip
+	$(VENV)/bin/pip install -r requirements.txt
+	$(VENV)/bin/pip install -r requirements-dev.txt
+	touch $(INSTALL_STAMP)
+
+format: $(INSTALL_STAMP)
+	$(VENV)/bin/isort . --virtual-env=$(VENV)
+	$(VENV)/bin/black kinto_remote_settings tests
+
+lint: $(INSTALL_STAMP)
+	$(VENV)/bin/isort . --check-only --virtual-env=$(VENV)
+	$(VENV)/bin/black --check kinto_remote_settings tests --diff
+	$(VENV)/bin/flake8 kinto_remote_settings tests
+
+test: $(INSTALL_STAMP) lint
+	PYTHONPATH=. $(VENV)/bin/pytest kinto_remote_settings
+
+integration-test: $(INSTALL_STAMP) lint need-kinto-running
+	docker-compose run tests
+
+need-kinto-running:
+	@curl http://localhost:8888/v1/ 2>/dev/null 1>&2 || (echo "Run 'make run-kinto' before starting tests." && exit 1)
```

Comment:
I suppose I could try checking the `/v1/__heartbeat__` endpoint to ensure we're getting back the expected fields?

Reply:
I thought about this some more. Since we run everything through Docker and docker-compose, maybe we can bake this check into the command we set for the `tests` service instead of having this `need-kinto-running` step. The Docker docs have some suggestions of how to accomplish this [here](https://docs.docker.com/compose/startup-order/).

The nice thing about this approach is that we'd be able to target the service running at `web:8888`, so we'll know that it's our kinto service.

Reply:
Seems like the docker compose file was already pretty much set for this to work, just needed to add an additional step in the integration test make command. https://github.com/mozilla-services/kinto-dist/pull/1953/commits/46ba22e4160f1f37c229065fe594aef695b87789